packages/0xcert-ethereum-proxy-contracts/src/contracts/xcert-update-proxy.sol

```diff
@@ -0,0 +1,39 @@
+pragma solidity 0.5.6;
+
+import "@0xcert/ethereum-xcert-contracts/src/contracts/ixcert-mutable.sol";
+import "@0xcert/ethereum-utils-contracts/src/contracts/permission/abilitable.sol";
+
+/**
+ * @title XcertUpdateProxy - updates a token on behalf of contracts that have been approved via
+ * decentralized governance.
+ * @notice There is a possibility of unintentional behavior when token imprint can be overwritten 
+ * if more than one claim is active. Be aware of this when implementing.
+ */
+contract XcertUpdateProxy is 
+  Abilitable 
+{
+
+  /**
+   * @dev List of abilities:
+   * 2 - Ability to execute create. 
+   */
+  uint8 constant ABILITY_TO_EXECUTE = 2;
+
+  /**
+   * @dev Updates imprint of an existing Xcert.
+   * @param _xcert Address of the Xcert contract on which the update will be perfomed.
+   * @param _id The Xcert we will update.
+   * @param _imprint Cryptographic asset imprint.
+   */
+  function update(
+    address _xcert,
+    uint256 _id,
+    bytes32 _imprint
+  )
```

Comment:
You probably meant ` _orderData.actions[i].to` right? Since value is used to hold the asset Id. But to is not used anywhere. It is irrelevant what is stored there as long as it is the same when signing and executing. In the framework I input the zero address (0x0000....).

Reply:
Yes. Perhaps we should specify that the value should be 0x0. Because 0x0 and any other value are valid and 0x0 costs less gas.

This is just a note and there is no need to delay this PR for it.